src/components/BurnIst.tsx

```diff
@@ -73,57 +73,48 @@ export default function BurnIst({ charterOfferId }: Props) {
       <h2 className="my-2 block text-lg leading-5 font-medium text-gray-700">
         Manual IST Burn
       </h2>
-      <motion.div
-        className="overflow-hidden px-1"
-        initial={{ height: 0, opacity: 0 }}
-        animate={{ height: 'auto', opacity: 1 }}
-        transition={{ type: 'tween' }}
-      >
-        <p className="my-1">{toBurnLabel}</p>
-        {brand && (
-          <AmountInput
-            disabled={isLoading}
-            value={currentInput}
-            brand={data?.shortfallBalance?.brand}
-            suffix="IST"
-            onChange={setCurrentInput}
-          ></AmountInput>
+      <p className="my-1">{toBurnLabel}</p>
+      <AmountInput
+        disabled={isLoading}
+        value={currentInput}
+        brand={data?.shortfallBalance?.brand}
+        suffix="IST"
+        onChange={setCurrentInput}
+      ></AmountInput>
```

Comment:
looks like this doesn't expect children
```suggestion
      <AmountInput
        disabled={isLoading}
        value={currentInput}
        brand={data?.shortfallBalance?.brand}
        suffix="IST"
        onChange={setCurrentInput}
      />
```

Reply:
Done